src/seedu/addressbook/common/Messages.java

```diff
@@ -23,6 +27,8 @@
     public static final String MESSAGE_USING_EXAMS_FILE = "Using exams file : %1$s";
     public static final String MESSAGE_EXAMS_LISTED_OVERVIEW = "%1$d exams listed!";
     public static final String MESSAGE_USING_STATISTICS_FILE = "Using statistics file : %1$s";
+    public static final String MESSAGE_STATISTICS_LISTED_OVERVIEW = "%1$d statistics listed!";
+    public static final String MESSAGE_INVALID_GRADES = "Invalid command format! - Value of grades is out of bounds!";
     public static final String MESSAGE_EXAM_NOT_IN_EXAMBOOK = "Exam could not be found in exam book!";
```

Comment:
Seems a bit bad naming. Out of bounds implies it is an index, but it says command format invalid